packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -248,6 +299,51 @@ contract L2ToL2CrossDomainMessengerTest is Test {
         );
     }
 
+    /// @dev Tests that the `relayMessage` function reverts if called by an address other than the specified entrypoint.
+    function testFuzz_relayMessage_wrongEntrypoint_reverts(
+        uint256 _source,
+        uint256 _nonce,
+        address _sender,
+        bytes calldata _message,
+        uint160 _entrypointSeed,
+        uint256 _value,
+        uint256 _blockNum,
+        uint256 _logIndex,
+        uint256 _time
+    )
+        external
+    {
+        // Expect a revert with the InvalidEntrypoint selector
+        vm.expectRevert(InvalidEntrypoint.selector);
+
+        address _target = makeAddr("target");
+
+        // Ensure entrypoint is not address(0) or Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER
+        _entrypointSeed = uint160(bound(uint256(_entrypointSeed), 1, type(uint160).max));
+        address _entrypoint = address(uint160(_entrypointSeed));
+
+        vm.assume(_entrypoint != Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER);
```

Comment:
Why not directly fuzzing the `_entrypoint` address assuming is neither `address(0)` or `L2_TO_L2_CROSS_DOMAIN_MESSENGER`?

Reply:
I thinks is better to avoid using assume since it will discard more runs.

Reply:
I saw that you applied this change hehe